### PR TITLE
Troubleshooting Documentation

### PR DESCRIPTION
**Pull request by akeeste**
This PR adds a Troubleshooting section to the User Manual. The first subsection covers:
- user expectations and steps to take before opening a GitHub issue
    - personal debugging expectations
    - identifying WEC-Sim vs MATLAB errors
    - reviewing relevant documentation, closed issues
    - opening a new issue (template, relevant information, uploading cases, tagging related issues, etc)
- issue board notes
    - developers can't address topics not directly related to WEC-Sim
    - timing expectations (<= 1 week for response)

The second subsection describes several numerical test cases users should follow when developing a novel case. This includes checking hydrostatic stability, decay tests, viscous drag, etc. For each test case, details are provided on typical errors and suggested solutions.

The third subsection contains a FAQ with a description, error message, and solutions for commonly asked github issues, including:
- solution singularity
- degenerate mass distribution
- h5 file

It it my hope that this page will set standards on using the issue board and help users address their own questions. It serves as a convenient place to point users with common support questions. Hopefully this decreases the time required to maintain the issue board. The most common issues can be added to the FAQ if desired. The debugging steps include reviewing past issues which should provide the same information.

### CODE REVIEW

**Comment by akeeste** [comment]
When building these docs there is an error:
"Extension error:
You must configure the bibtex_bibfiles setting"

I'll look into and see if the new .rst file is causing any issues.

File: docs/user/troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+Test 3:
+^^^^^^^
+
+A hydrostatically stable device that has an unphysical response to a regular wave requires different drag and damping.
+Viscous drag is essential to a physical response in WEC-Sim.
+Tune the parameters ``body(#).viscDrag`` or ``body(#).linearDamping`` to cause a more realistic response.
+
+Test 4:
+^^^^^^^
+
```
Comment (jtgrasb) [inline docs/user/troubleshooting.rst:143]: Any suggestions if it fails test 4?

File: docs/user/troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+.. code-block:: none
+
+	... Joint has a degenerate mass distribution on its base/follower side.
+
+Solution
+~~~~~~~~
+
+Add an insignificantly small mass (e.g. ``Simulink Library/Simscape/Multibody/Body Elements/Inertia``) between the two joints.
+Alternatively, if special degrees of freedom are required create a new PTO or constraint using one of the many joints
```
Comment (jtgrasb) [inline docs/user/troubleshooting.rst:238]:
```suggestion
Alternatively, if special degrees of freedom are required, create a new PTO or constraint using one of the many joints 
```

**Review by jtgrasb — COMMENTED** [review]
Just small, grammatical changes. Overall, should be very helpful for new users and looks good once the building error is fixed.

File: docs/user/troubleshooting.rst (hunk 1 above)
Comment (akeeste) [reply docs/user/troubleshooting.rst:144]: Good point, the tests get more difficult to debug as they get more complicated. I will put in a note for users to check the IRF in BEMIO as for the 'B' tests. Also users can try the state space, or changing stiffness, damping, etc. I'll make notes of this

**Comment by akeeste** [comment]
@Jeffalo1 I updated the new page based on your suggestions. I also pulled dev into this branch so that the docs build properly now.

**Pull request merged**